apps/prairielearn/src/sprocs/instance_questions_grade.sql

```diff
@@ -17,10 +17,20 @@ BEGIN
 
     IF NOT instance_question_open THEN
         -- this shouldn't happen, so log an error
+        --
+        -- TODO: We should actually work to prevent this from happening
+        -- farther upstream, and avoid recording an issue here.
         PERFORM issues_insert_for_variant(
-            v.id, 'Submission when instance question is closed', '', false,
-            false, jsonb_build_object('grading_job_id', grading_job_id),
-            '{}'::jsonb, instance_questions_grade.authn_user_id)
+            v.id,
+            'Submission when instance question is closed',
+            '',
+            false,
+            false,
+            jsonb_build_object('grading_job_id', grading_job_id),
+            '{}'::jsonb,
+            instance_questions_grade.authn_user_id,
+            instance_questions_grade.authn_user_id
```

Comment:
Nit: some comments on what these arguments represent would be useful.

Reply:
Done in e3b1924, look good?

Reply:
The last two were not the problem, it's the blanks and falses that are confusing. That said, I understand this is an issue that was there before, so feel free to ignore it.

Reply:
For future spelunkers, see https://github.com/PrairieLearn/PrairieLearn/pull/10946#discussion_r1859036166 for more context.